### ISSUE TITLE
Pages in PageView are not clickable when scrolling

### ISSUE THREAD

**Issue by ksheremet** [flutter/flutter#39984]
## Steps to Reproduce

During scrolling, pages in PageView widget
 are not clickable. The gist is provided here: 

https://gist.github.com/ksheremet/f4831b3c1bc6e31235caca5dd836b245 .

![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/6387464/64442702-90a4b880-d0d0-11e9-8b78-c07fd4522513.gif)

```
[✓] Flutter (Channel dev, v1.9.7, on Mac OS X 10.14.6 18G87, locale en-CH)
    • Flutter version 1.9.7 at /Users/ksheremet/pkg/flutter
    • Framework revision 4984d1a33d (9 days ago), 2019-08-28 17:04:07 -0700
    • Engine revision f52c0b9270
    • Dart version 2.5.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    • Android SDK at /Users/ksheremet/pkg/android
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.0
    • ANDROID_HOME = /Users/ksheremet/pkg/android
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 10.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.3, Build version 10G8
    • CocoaPods version 1.7.0

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 37.1.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[✓] IntelliJ IDEA Community Edition (version 2019.1.4)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 38.2.3
    • Dart plugin version 191.8369

[✓] VS Code (version 1.38.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.4.1

[✓] Connected device (1 available)
    • Pixel 3a • 93RAY0EK2R • android-arm64 • Android 9 (API 28)

• No issues found!
```

**Comment by goderbauer** [comment]
That is intentional because tapping something that's moving makes it very likely that your finger doesn't hit the right target (usually the tap targets are way smaller than in your example).

**Comment by tp** [comment]
> That is intentional because tapping something that's moving makes it very likely that your finger doesn't hit the right target

@goderbauer: I can understand that reasoning, but don't think it applies here.

In @ksheremet's example (e.g. tap 2) and also an internal horizontal `PageView` we see this behavior where tapping is not possible, even though the list is "visually at rest".

So I think there is still an issue with the timing / threshold of when taps are enabled again and I would ask you to consider reopening this.

**Comment by goderbauer** [comment]
If you have a case where tapping is not possible even though it should because the scroll is at rest, I'd recommend opening a new issue with code that demonstrates the problem.

From the video seen above it looks to me as if the scroll is still in progress whenever the tap happens.

**Comment by tp** [comment]
> From the video seen above it looks to me as if the scroll is still in progress whenever the tap happens.

That aligns with our experience, tapping is not possible when it's scrolling.

Hence I used "visually at rest" to describe our case. In our testing we found users want to tap things that were still moving – at a very slow pace but still perceptible pace (snapping back into place in a `PageView`).

With the current behavior they'll have to tap twice, as the first tap won't do anything and by the time they notice and tap again it'll work.

I'll accept this as closed / "works as designed" for now. If this gets reported again I would open a new issue with our concrete screen (as I guess the concrete UI also makes a difference as to what seems to be "expected").
